### PR TITLE
LPS-193879 Add basic Actions UI without Icons or Localized Inputs

### PR DESCRIPTION
**Pull request by kresimir-coko**
## How it looks

image

### CODE REVIEW

**Comment by kresimir-coko** [comment]
ci:test:sf

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Actions.tsx [hunk 1]
```diff
@@ -3,29 +3,338 @@
...
+										},
+										{
+											label: Liferay.Language.get('type'),
+											name: 'type',
+										},
+									]}
+									items={[]}
+									noItemsButtonLabel={Liferay.Language.get(
+										'create-item-action'
+									)}
```
Comment (markocikos) [inline modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Actions.tsx:133]: Not required in this task, but we should apply to the `OrdereableTable` the same `emptyState` API that we recently added to FDS.

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Actions.tsx [hunk 2]
```diff
@@ -3,29 +3,338 @@
...
+
+							<ClayTabs.TabPane aria-labelledby="tab-2">
+								2. Proin efficitur imperdiet dolor, a iaculis
+								orci lacinia eu.
+							</ClayTabs.TabPane>
+
+							<ClayTabs.TabPane aria-labelledby="tab-3">
+								3. Proin efficitur imperdiet dolor, a iaculis
+								orci lacinia eu.
+							</ClayTabs.TabPane>
```
Comment (markocikos) [inline modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Actions.tsx:157]: I would not even put a placeholder for bulk actions here. It is planned for Q4 I believe, so we will need to delete this later on or put behind new FF :grimacing: I also don't see much sense in bulk actions as a separate configuration category, they are basically item actions. We need to add support for them in FDS, sure, but I don't see any configuration additional to item actions that we would need to add here.

**Review by markocikos — CHANGES_REQUESTED** [review]
@kresimir-coko Mostly looks good! Some minor stuff in comments.

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Actions.tsx (hunk 1 above)
Comment (markocikos) [reply modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Actions.tsx:146]: I opened a task for this: https://liferay.atlassian.net/browse/LPS-194347

**Review by markocikos — APPROVED** [review]
@kresimir-coko LGTM! :sparkles:

**Comment by kresimir-coko** [comment]
ci:test:sf

**Comment by kresimir-coko** [comment]
ci:test:relevant

**Comment by kresimir-coko** [comment]
ci:test:relevant

**Comment by markocikos** [comment]
ci:forward

**Comment by thektan** [comment]
Manually forwarded https://github.com/brianchandotcom/liferay-portal/pull/139762

**Comment by thektan** [comment]
ci:close

**Comment by markocikos** [comment]
ci:forward

**Comment by markocikos** [comment]
oops, forwarded wrong PR :man_facepalming:

**Pull request closed without merging**